openedx_webhooks/templates/github_community_pr_comment.md.j2

```diff
@@ -22,9 +22,16 @@ GitHub pull request interface. As a reminder,
 
 {% filter replace("\n", " ")|trim %}
   {%- if not has_signed_agreement %}
-    We can't start reviewing your pull request until you've submitted a
+    We can't start reviewing your pull request until you and the other commit authors submit a
```

Comment:
I wonder if we need to make this sentence more sophisticated, so that it reads properly in all cases.